02LessonTwo/test/LessonTwoTest.js.orig

```diff
@@ -0,0 +1,43 @@
+var assert = require('assert');
```

Comment:
Cool , I think i just did it correctly

On Wed, Sep 23, 2015 at 11:52 AM Kevin Colten <notifications@github.com>
wrote:

> In 02LessonTwo/test/LessonTwoTest.js.orig
> <https://github.com/AustinCodingAcademy/HTMLIntermediateSection2Tests/pull/6#discussion_r40227755>
> :
>
> > @@ -0,0 +1,43 @@
> > +var assert = require('assert');
>
> See if you can delete this file. You should be able to delete on your
> machine, the commit the change, and push it up. Then it should disappear
> from here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AustinCodingAcademy/HTMLIntermediateSection2Tests/pull/6/files#r40227755>
> .
>
-- 
Peter Arevalo